mne/connectivity/tests/test_envelope.py

```diff
@@ -15,19 +15,23 @@
 def _compute_corrs_orig(data):
     # This is the version of the code by Sheraz and Denis.
     # For this version (epochs, labels, time) must be -> (labels, time, epochs)
-    data = np.transpose(data, (1, 2, 0))
-    corr_mats = np.empty((data.shape[0], data.shape[0], data.shape[2]))
-    for index, label_data in enumerate(data):
-        label_data_orth = np.imag(label_data * (data.conj() / np.abs(data)))
-        label_data_orig = np.abs(label_data)
-        label_data_cont = np.transpose(
-            np.dstack((label_data_orig, np.transpose(label_data_orth,
-                                                     (1, 2, 0)))), (1, 2, 0))
-        corr_mats[index] = np.array([np.corrcoef(dat)
-                                     for dat in label_data_cont])[:, 0, 1:].T
-    corr_mats = np.transpose(corr_mats, (2, 0, 1))
-    corr = np.mean(np.array([(np.abs(corr_mat) + np.abs(corr_mat).T) / 2.
-                             for corr_mat in corr_mats]), axis=0)
```

Comment:
>  I always had a feeling since our working session at Biomag2018 that the MNE implementation does something different than the prototype.

@dengemann IIRC the code here was the prototype that I used to code a more efficient version in MNE, so it should have replicated it properly. Unfortunately I had to change this prototype code to do something different in order to get it to follow FieldTrip and the paper definitions...

Reply:
No problem, If we can come up with a more flexible set of options this is good for research on this method.
The comparisons in the previous discussion suggest that difference may be rather subtle.

Reply:
I second, differences will be subtle.